### PR TITLE
Exo sos ls - add raw listing

### LINKED ISSUE #29: Add a `--vanilla` flag to `exo sos list`
Listing objects in buckets is great but gives this kind of output:

     [2018-06-28 12:17:19 UTC] 411 KiB FILENAME_1
     [2018-06-28 12:17:18 UTC] 418 KiB FILENAME_2
     ...

It would be nice to have an option to have the output as filename only, in order to use the list in subsequent commands without having to manipulate the string.

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Add `-R`, `--raw` on `exo sos ls` raw display of listing
Fix #29

### CODE REVIEW

File: cmd/sos_list.go [hunk 1]
```diff
@@ -111,6 +113,11 @@ var sosListCmd = &cobra.Command{
 			key := filepath.ToSlash(message.Key)
 			key = strings.TrimLeft(key[len(prefix):], "/")
 
+			if isRaw {
+				fmt.Fprintf(table, "%s\n", key) // nolint: errcheck
```
Comment (greut) [inline cmd/sos_list.go]:
```suggestion
				fmt.Fprintln(table, key) // nolint: errcheck
```

File: cmd/sos_list.go [hunk 2]
```diff
@@ -132,19 +139,27 @@ func listRecursively(c *minio.Client, bucketName, prefix, zone string, displayBu
...
+				fmt.Fprintf(table, "%s/%s\n", bucketName, message.Key) // nolint: errcheck
+				continue
+			}
 			fmt.Fprintf(table, "%s\t%s\t%s\t%s\n", fmt.Sprintf("[%s]", lastModified), // nolint: errcheck
 				fmt.Sprintf("[%s]", zone),
 				fmt.Sprintf("%6s ", humanize.IBytes(uint64(message.Size))),
 				fmt.Sprintf("%s/%s", bucketName, message.Key)) // nolint: errcheck
 		} else {
+			if isRaw {
+				fmt.Fprintf(table, "%s\n", message.Key) // nolint: errcheck
```
Comment (greut) [inline cmd/sos_list.go]:
```suggestion
				fmt.Fprintln(table, message.Key) // nolint: errcheck
```

File: cmd/sos_list.go [hunk 3]
```diff
@@ -132,19 +139,27 @@ func listRecursively(c *minio.Client, bucketName, prefix, zone string, displayBu
...
 				fmt.Sprintf("%s/%s", bucketName, message.Key)) // nolint: errcheck
 		} else {
+			if isRaw {
+				fmt.Fprintln(table, message.Key) // nolint: errcheck
+				continue
+			}
 			fmt.Fprintf(table, "%s\t%s\t%s\n", fmt.Sprintf("[%s]", lastModified), // nolint: errcheck
 				fmt.Sprintf("%6s ", humanize.IBytes(uint64(message.Size))),
-				fmt.Sprintf("%s/%s", bucketName, message.Key)) // nolint: errcheck
+				message.Key) // nolint: errcheck
```
Comment (greut) [inline cmd/sos_list.go]:
```go
key := message.Key
if displayBucket {
    key = fmt.Sprintf("%s/%s", bucketName, key)
}
```

File: cmd/sos_list.go [hunk 4]
```diff
@@ -169,34 +163,19 @@ func displayBucket(minioClient *minio.Client, isRecursive, isRaw bool) error {
 
 	for zoneName, buckets := range allBuckets {
 		for _, bucket := range buckets {
+			if isRaw {
+				fmt.Fprintf(table, "%s/\n", bucket.Name) // nolint: errcheck
+			} else {
+				fmt.Fprintln(table,
+					fmt.Sprintf("[%s]\t[%s]\t%6s \t%s/", bucket.CreationDate.Format(printDate), zoneName, humanize.IBytes(uint64(0)), bucket.Name)) // nolint: errcheck
```
Comment (greut) [inline cmd/sos_list.go]: Meme quiz

Comment (pierre-emmanuelJ) [reply cmd/sos_list.go]: 😹 😂 ohww my bad

**Comment by greut** [comment]
@pierre-emmanuelJ please review the last commit before merging. :+1:

**Comment by pierre-emmanuelJ** [comment]
@greut thanks but I have to fix an other formating display

**Pull request merged**